lib/src/magic_number.dart

```diff
@@ -122,14 +122,500 @@ const List<MagicNumber> initialMagicNumbers = [
     0x74,
     0x79,
     0x70,
-    0x33,
```

Comment:
This looks like it removes the version with the last 4 bytes `[0x33, 0x67, 0x70, 0x35]` or `3gp5`

According to https://www.ftyps.com this mime type should be `video/3gpp` though - so I don't know if it was useful when it was mapped to `video/mp4`. I think it's OK to overwrite this old entry with the new one.

Looking at just the types that are marked as registered and map to `video/mp4` it looks fairly close to your list. However there are some differences.

- `mp71` does not link directly to the mime type `video/mp4`, this resource lists it as "various" - maybe it's OK to include anyway?
 - `msnv` is capitalized as `MSNV` and maps to `audio/mp4`
 - `ndsc` through `ndxs` are all capitalized, and none are marked as registered.
 
 I think we can probably prune the list down to `avc1`, `iso2`, `isom`, `mp41`, and `mp42`

Reply:
I returned the type "3gp5 = 33 67 70 35" and marked it as video/3gpp. And also I pruned  the MP4 list down to avc1, iso2, isom, mp41 and mp42.